src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs

```diff
@@ -123,6 +123,8 @@ internal sealed class ServiceFunction : BuiltinFunction
 
                 var optionFormat = new StringBuilder(TexlLexer.PunctuatorCurlyOpen);
                 string sep = "";
+
+                // $$$ can't use current culture
```

Comment:
>// $$$ can't use current culture [](http://example.com/codeflow?start=16&length=32)

There are lots of instances of this comment - we should have a task / issue tracking this

Reply:
We have one to remove all $$$ comments
We'll create a specific one for current culture